apps/fair/client/index.coffee

```diff
@@ -1,9 +1,9 @@
 _ = require 'underscore'
 Backbone = require 'backbone'
 sd = require('sharify').data
+MarketingSignupModal = require '../../../components/marketing_signup_modal/index.coffee'
 initFairLayout = require '../../../components/fair_layout/client/index.coffee'
 FairBrowseRouter = require '../components/browse/router.coffee'
-# FairInfoView = require './info.coffee'
```

Comment:
I removed these commented lines, I think it was accidentally left from someone else.